src/slack-integration.sh

```diff
@@ -10,10 +10,13 @@ throw() {
 [ -z "$SLACK_CI_CHANNEL_RELEASE" ] && throw 'Environment variable '"$SLACK_CI_CHANNEL_RELEASE" 'is not set.'
 [ -z "$SLACK_CI_CHANNEL_DEV" ] && throw 'Environment variable '"$SLACK_CI_CHANNEL_DEV" 'is not set.'
 
-channel="$SLACK_CI_CHANNEL_DEV"
-
 if [ "$CIRCLE_BRANCH" = "master" ] || [ "$CIRCLE_BRANCH" = "release" ]; then
```

Comment:
FYI: Extension has a different set of release branches. `release-axe` and `release-attest`. We do this because different pipelines kick off depending on which branch it is.

Reply:
Should I do something like 
```suggestion
if [ "$CIRCLE_BRANCH" = "master" ] || [ "$CIRCLE_BRANCH" = "release" ] || [ "$CIRCLE_BRANCH" = "release-axe" ] || [ "$CIRCLE_BRANCH" = "release-attest" ]; then
```

because if I do something like `/^release-*/` any release candidate could send a msg to slack which would make more noise in the channel than needed

Reply:
Can release branch be an environment var that defaults to `"release"`? 

Also I'm scope creeping so feel free to create another issue for this.